checker/tests/nullness/Issue1142.java

```diff
@@ -0,0 +1,14 @@
+// Issue 1142 https://github.com/typetools/checker-framework/issues/67
+import java.util.concurrent.ConcurrentHashMap;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class Issue1142 {
+
+    void foo() {
+
+        ConcurrentHashMap<Integer, @Nullable Integer> chm = new ConcurrentHashMap<>();
```

Comment:
Yes Professor,
Currently this line is throwing an error, stating:
found: @Initialized @Nullable
required: @Initialized @NonNull